contracts/Rules.sol

```diff
@@ -173,8 +116,7 @@ contract Rules is AdminProxy, RulesProxy {
         bytes32 enodeLow,
         bytes16 enodeIp,
         uint16 enodePort
-    ) public onlyAdmin returns (bool) {
-        require(readOnlyMode == false, "In read only mode: rules cannot be modified");
+    ) public onlyAdmin onlyOnEditMode returns (bool) {
```

Comment:
Don't think this reference of onlyAdmin is calling to the admin contract, think it's calling to an onlyAdmin thats meant to be defined on the rules.

Reply:
I believe it is executing the proper modifier in the Admin contract. At least if I change the modifier on the Admin contract and run the Rules test, they all start failing.

Reply:
I don't know how it's figuring out what address the admin contract is deployed at.

Reply:
it's importing it

Reply:
It isn't. I'm changing it to use a "remote" call instead of embedding the Admin contract into the Rules contract.

Reply:
👍